From 45ced7cd7ea302bc9c4a61e1d328b6b0a8d52915 Mon Sep 17 00:00:00 2001
From: SignKirigami <prcups@krgm.moe>
Date: Fri, 19 Jan 2024 12:24:37 +0800
Subject: [PATCH] update gotools

A dependency of gotools requires high version to support loongarch. Pull
Request has been merged but gotools should be manually updated before next main
version.

---
 build/update_pkg.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/update_pkg.sh b/build/update_pkg.sh
index b6d9de9b..53bd0fdd 100755
--- a/build/update_pkg.sh
+++ b/build/update_pkg.sh
@@ -25,7 +25,7 @@ export GOBIN=$PWD/bin
 echo install gocode ...
 go install -v github.com/visualfc/gocode@latest
 echo install gotools ...
-go install -v github.com/visualfc/gotools@latest
+go install -v github.com/visualfc/gotools@45f301a
 echo install gomodifytags ...
 go install -v github.com/fatih/gomodifytags@latest
 
-- 
2.39.1

